alien4cloud/deployment.go

```diff
@@ -33,6 +38,10 @@ type DeploymentService interface {
 	DeployApplication(ctx context.Context, appID string, envID string, location string) error
 	// Updates an application with the latest topology version
 	UpdateApplication(ctx context.Context, appID, envID string) error
+	// Updates inputs of a deployment topology
+	UpdateDeploymentSetup(ctx context.Context, appID, envID string, request UpdateDeploymentTopologyRequest) error
```

Comment:
Why `Setup`?

Reply:
To use the same name as the Alien4Cloud API we are using, at https://github.com/alien4cloud/alien4cloud/blob/2.2.0-SM10/alien4cloud-rest-api/src/main/java/alien4cloud/rest/deployment/DeploymentTopologyController.java#L344

Reply:
Ok but the REST endpoint name is [`deployment-topology`](https://github.com/alien4cloud/alien4cloud/blob/2.2.0-SM10/alien4cloud-rest-api/src/main/java/alien4cloud/rest/deployment/DeploymentTopologyController.java#L57).
I think we should stick on Rest API naming instead of java code naming. WDYT?

Reply:
ok, I will replace Setup by Topology